AI #1613: Change ChargeType reference to ChargeCategory

shawnalpay opened the pull request.

shawnalpay committed:
fix chargetype reference

mike-finopsorg committed:
AI #1600: Set working_draft github workflow not to run on candidate_recommendation branch (#1601)

jpradocueva commented:
Approved by the group without objections or abstentions during the November 13 Members’ call.
TO BE MERGED ON DECEMBER 4th AT THE END OF THE IPR REVIEW

shawnalpay committed:
AI #1618: Repair value format references (#1619)

A couple of consistency fixes for the value formats.

---------

Co-authored-by: Joaquin <jprado@linuxfoundation.org>

ijurica committed:
AI #1631: Address Minor Editorial Issues in FOCUS 1.3 (#1632)

This PR addresses minor editorial issues in the FOCUS 1.3, as outlined
in Action Item #1631

shawnalpay committed:
AI #1530: Revise Status values (#1585)

We have added a new entry to the Status attribute of the FOCUS WG
project view (`In Evaluation`) to represent the Feature Request Review
process we have introduced to FOCUS 1.4.

This PR updates the documentation for the Status attribute in the
development processes guidelines.

This PR is NOT to be approved and merged as part of 1.3, as it was not
part of that release process. We can approve and merge in any Members
meeting after Nov 6.

---------

Co-authored-by: Joaquin Prado <joaquin@elastichub.io>
Co-authored-by: Joaquin <jprado@linuxfoundation.org>

jpradocueva reviewed (approved):
Approved by the Members on the November 13th call.

mike-finopsorg committed:
Fix #1608 EffectiveCost-C-004-M has incorrectly defined requirements (#1609)

Add unittest to ensure no other similar issues found and to keep this
happening again
Signed-off-by: Mike Fuller <mike@finops.org>

Co-authored-by: Joaquin <jprado@linuxfoundation.org>

jpradocueva committed:
Merge branch 'working_draft' into 1613_repair_chargetype_reference

jpradocueva committed:
Merge branch 'candidate_recommendation' into 1613_repair_chargetype_reference
